2021 01 19 shaun liu pim (#11399)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/unifiedrolemanagementpolicynotificationrule.md

---
title: "unifiedRoleManagementPolicyNotificationRule resource type"

> description: "A unifiedRoleManagementPolicyNotificationRule specifies the notification rule associated with a role management policy. It is derived from microsoft.graph.unifiedRoleManagementPolicyRule."

Reviewer: ```suggestion
description: "A unifiedRoleManagementPolicyNotificationRule specifies the notification rule associated with a role management policy."
```

author: "shauliu"
localization_priority: Normal
ms.prod: "microsoft-identity-platform"
doc_type: resourcePageType
---

# unifiedRoleManagementPolicyNotificationRule resource type

> Namespace: microsoft.graph

Reviewer: Add: [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> A unifiedRoleManagementPolicyNotificationRule specifies the notification rule associated with a role management policy. It is derived from microsoft.graph.unifiedRoleManagementPolicyRule.

Reviewer: ```suggestion
A unifiedRoleManagementPolicyNotificationRule specifies the notification rule associated with a role management policy.
```

Inherits from [unifiedRoleManagementPolicyRule](../resources/unifiedrolemanagementpolicyrule.md).

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |id|String|Unique identifier for the rule. Inherited from [unifiedRoleManagementPolicyRule](../resources/unifiedrolemanagementpolicyrule.md)|

Reviewer: ```suggestion
|id|String|Unique identifier for the rule. Inherited from [unifiedRoleManagementPolicyRule](../resources/unifiedrolemanagementpolicyrule.md).|
```

> |notificationLevel|String|The level of notification. One of None, Critical, All.|

Reviewer: ```suggestion
|notificationLevel|String|The level of notification. Possible values: `None`, `Critical`, and `All`.|
```

> |notificationRecipients|String collection|The list of notification recepients like email.|

Reviewer: ```suggestion
|notificationRecipients|String collection|The list of notification recipients, such as email.|
```

> |notificationType|String|The type of notification. One of Email.|

Reviewer: ```suggestion
|notificationType|String|The type of notification. The value must be `Email`.|
```

> |recipientType|String|The type of recipient. One of Requestor, Approver, Admin.|

Reviewer: ```suggestion
|recipientType|String|The type of recipient. Possible values: `Requestor`, `Approver`, and `Admin`.|
```

> |target|[unifiedRoleManagementPolicyRuleTarget](../resources/unifiedrolemanagementpolicyruletarget.md)|The target for the rule. Inherited from [unifiedRoleManagementPolicyRule](../resources/unifiedrolemanagementpolicyrule.md)|

Reviewer: ```suggestion
|target|[unifiedRoleManagementPolicyRuleTarget](../resources/unifiedrolemanagementpolicyruletarget.md)|The target for the rule. Inherited from [unifiedRoleManagementPolicyRule](../resources/unifiedrolemanagementpolicyrule.md).|
```
